content/en/docs/reference/_index.md

```diff
@@ -63,6 +67,15 @@ client libraries:
 * [Scheduler Profiles](/docs/reference/scheduling/config#profiles)
 
 
+## Config APIs
+
+This section hosts the documentation for "unpublished" APIs which are used to
+configure the behavior for kubernetes components or tools. Most of these APIs
```

Comment:
nit: This section hosts the documentation for "unpublished" APIs which are used to
configure Kubernetes components or tools.

Reply:
Done.